src/core/cancel-token.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {CancelToken} from 'cancel-token';
+
+/**
+ * The analyzer only uses one method from CancelTokens, and it's likely to be
+ * the least contentious, so even if other parts change on the way to
+ * standardization,
```

Comment:
standardization? :)

Reply:
Weird looking, but I think that's the right spelling: standardization